core/src/main/java/org/kohsuke/stapler/export/Property.java

```diff
@@ -168,6 +175,19 @@ private void writeValue(Type expected, Object value, TreePruner pruner, DataWrit
         writeValue(expected,value,pruner,writer,false);
     }
 
+    private void writeBuffered(Type expected, Object value, TreePruner pruner, DataWriter writer) throws IOException {
+        BufferedDataWriter buffer = new BufferedDataWriter();
+        try {
+            writeValue(expected, value, pruner, buffer, true);
+            buffer.finished();
+        } catch (IOException x) {
+            if (x.getCause() instanceof InvocationTargetException) {
+                LOGGER.log(Level.WARNING, "skipping export of " + value, x);
```

Comment:
I doubt it. #48 downgraded a message printed after upgrading some components against others which were still using what used to be a slightly unusual but otherwise supported construction.

The case fixed here is limited to genuine plugin bugs, generally ones which are trivial to diagnose and assign when encountered; and either trivial to fix completely, or at least ameliorate by catching in the `@Exported` method.

And again we already have tons of code in Jenkins core which catches a broad range of exceptions from extensions or other plugin code, logs it, and continues, including code paths which might be encountered at a high frequency depending on usage patterns. Logging these things at `FINE` will just mean that the bugs are never fixed. If there is a persistent issue with logs filling up with hundreds of more or less identical stack traces due to buggy plugins, that is best handled by the logging system, for example by printing a message saying that a given `Logger` has produced too much noise and it is going to be suppressed for the rest of the session.